src/types/global.d.ts

```diff
@@ -3,3 +3,9 @@ declare module "*.json" {
   const value: any
   export default value
 }
+
```

Comment:
Global declarations for json imports were broken since we were not referencing to the types correctly. Please read this for more details: https://mariusschulz.com/articles/importing-json-modules-in-typescript